### DISCUSSION TITLE
Investigation advice

### DISCUSSION THREAD

**Discussion by kehtabp** [kamaravichow/safe-dot-android#27] [Q&A]
So when I run relay for Reddit I get a camera permission in use dot every minute or 2. It's weird since it doesn't have camera permission. 

I understand the app just reports the foreground app, but it's weird that it mostly happens with relay and not the other apps. I would like to understand what is going on in the background. 

Any advice on how to go about investigating this?

**Comment by kamaravichow** [comment]
Hmm, that's weird. I'll try to find out the reason and get back to you.
